src/components/PostList/PostList.jsx

```diff
@@ -1 +1,10 @@
-export const PostList = () => <>Put the list here</>;
+import { PostInfo } from '../PostInfo/PostInfo';
+import './PostList.scss';
+
+export const PostList = ({ post }) => (
+  <div className="PostList">
+    {post.map(postItem => {
+      return <PostInfo post={postItem} key={postItem.id} />;
+    })}
```

Comment:
```suggestion
    {post.map(postItem => (
      <PostInfo post={postItem} key={postItem.id} />;
    ))}
```
FYI: you may write it in the shorter way